openmoc/map_to_dict.i

```diff
@@ -96,10 +96,10 @@
 }
 
 
-/* Typemap for Lattice::setUniverses(int num_x, int num_y, Universe** universes)
+/* Typemap for Lattice::setUniverses(int num_y, int num_x, Universe** universes)
```

Comment:
There are three locations where the setUniverses method needs to be defined: map_to_dict.i, Universe.h, and Universe.cpp. There was an inconsistency in the current develop branch on the method definitions, so this change and a change in Universe.h makes them consistent.